tests/unit_tests/expected_md/details_error.md

```diff
@@ -0,0 +1,3 @@
+Error: `ValueError('Environment variable CONBENCH_URL not found')`
+
+See build link below.
```

Comment:
I presume in a real case, there would be link(s) below?

Reply:
Yes -- the check webpage shows the title, then summary, then a "DETAILS" section in its own little box, then a "View additional details" link which is the URL.